Dockerfile.singlestage

```diff
@@ -0,0 +1,38 @@
+# For more information, please refer to https://aka.ms/vscode-docker-python
```

Comment:
Delete this file. No reason to keep it.

Reply:
I wanted to keep this file so I can see the difference between single stage and multi-stage docker builds.